docs/advanced/Testing.md

```diff
@@ -1,9 +1,35 @@
 # Examples of Testing Sagas
 
-**WIP**
+There is ongoing discussion in [this issue](https://github.com/redux-saga/redux-saga/issues/518) about the best way to test Sagas.
 
-See Repository Examples:
+
+In the repo we also have two examples showing how Sagas can be tested:
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/counter/test/sagas.js
 
 https://github.com/redux-saga/redux-saga/blob/master/examples/shopping-cart/test/sagas.js
+
+
+
+***External Resources***
+
+Redux Saga Test Plan - https://github.com/jfairbank/redux-saga-test-plan
+
+From the README:
```

Comment:
Im not sure including external READMEs is the good idea, cause its really easy for them to get stale once original repos get updated.

Also the included README for Redux Saga Test Plan is much longer than the one for Redux Saga Test which might give the impression its the preferred way etc (people tend to judge based on such criteria really often).

I think I would keep only the links to those repos with a NOTE above that those are community attempts to provide a more robust testing tools, but that we do not favor one over another and also we do not recommend using them per se. It is up to individual developer to chose.

WDYT?